### PR TITLE
444 program minor fixes

### PR DESCRIPTION
**Pull request by Castronova**
* fixed typo on model program landing page
* added a check for https in modelinstance.html to ensure the model program resource path is correct.
* removed class name **MpModelProgram** from model program edit page
* fixed issue that was preventing files from being uploaded after model program and model instance are created.
* Changed *Unknown* to *Unspecified* when selecting a program on the instance edit page.
* Displaying *Unspecified* for program on instance landing page, if no program has been selected.

### CODE REVIEW

**Comment by pkdash** [comment]
+1

File: hs_model_program/models.py [hunk 1]
```diff
@@ -110,6 +110,10 @@ def can_delete(self, request):
     def can_view(self, request):
         return AbstractResource.can_view(self, request)
 
+    @classmethod
```
Comment (Castronova) [inline hs_model_program/models.py]: This change allows files to be uploaded after the resource has already been created.  Without this, an error prevents file upload.

**Comment by Castronova** [comment]
@mohamedmorsy @pkdash  I cannot get this to work...making the suggested change breaks model instance resource creation.  Probably because executedby **model_name** is a choice box.  In addition, I don't think this will simplify the if-else in modelinstanceresource.html because we would still need to check if model version and landing page exist. 

I suggest that we skip this for now and work it into release 1.6.

**Comment by pkdash** [comment]
@Castronova I agree. Since we don't have time to make this improvement/refactor now, you can create an issue in github for this to be resolved for release 1.6. It should be merged then.

**Comment by mohamedmorsy** [comment]
+1

**Pull request merged**